### PR TITLE
Statistics extractor refactoring

### PR DESCRIPTION
**Pull request by pbylicki**
@MaciejWiczk Some refactoring to illustrate how we could use shared model between API and client code.

### CODE REVIEW

File: rfhub2/cli/statistics_importer.py [hunk 1]
```diff
@@ -1,82 +1,83 @@
...
-        This is simplified approach to quick check file.
-        :param path:
-        :return:
-        """
-        with open(path, "r", encoding="utf-8", errors="ignore") as f:
-            # read the first few lines; if we don't see
-            # what looks like robot generator tag, return false
-            data = f.read(200)
-            return "<robot generator=" in data.lower()
+from pathlib import Path
```
Comment (MaciejWiczk) [inline rfhub2/cli/statistics_importer.py:1]: Anything happened to this file or is it just Win/Lin conversion?

**Review by MaciejWiczk — APPROVED** [review]
image

**Comment by MaciejWiczk** [comment]
> @MaciejWiczk Some refactoring to illustrate how we could use shared model between API and client code.

Is it part of #31?

File: rfhub2/cli/statistics_importer.py (hunk 1 above)
Comment (pbylicki) [reply rfhub2/cli/statistics_importer.py:1]: only meaningful change is updated type annotation in L48

**Comment by pbylicki** [comment]
> > @MaciejWiczk Some refactoring to illustrate how we could use shared model between API and client code.
> 
> Is it part of #31?

It is a step in that direction :) #31 will require doing this for keywords and collections with some refactorings available when all components follow the same approach.

**Comment by MaciejWiczk** [comment]
> > > @MaciejWiczk Some refactoring to illustrate how we could use shared model between API and client code.
> > 
> > 
> > Is it part of #31?
> 
> It is a step in that direction :) #31 will require doing this for keywords and collections with some refactorings available when all components follow the same approach.

I will create PR with some changes for keywords and collections, now that I know how it should be done.

File: rfhub2/cli/statistics_importer.py (hunk 1 above)
Comment (MaciejWiczk) [reply rfhub2/cli/statistics_importer.py:1]: GitHub was just showing that all file has changed. Strange

**Pull request merged**